v3/src/index.scss

```diff
@@ -11,12 +11,12 @@ body {
   }
 
   /* Show outline for keyboard :focus-visible */
-  .chakra-button {
-    &:focus-visible {
-      // chakra uses box-shadow for focus highlight rather than outline
-      box-shadow: var(--chakra-shadows-outline) !important;
-    }
-  }
+  // .chakra-button {
+  //   &:focus-visible {
+  //     // chakra uses box-shadow for focus highlight rather than outline
+  //     box-shadow: var(--chakra-shadows-outline) !important;
+  //   }
+  // }
```

Comment:
Is this change related to this PR? This was done to prevent distracting highlighting when clicking while not disabling highlighting for keyboard focus. Do you have reason to believe it's no longer necessary?

Reply:
Highlighting wasn't coming up on keyboard focus. But you're right, it shouldn't be part of this PR